### PR TITLE
{lang}[foss/2017b] Ruby v2.4.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2622.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0d23ecc951ab03256623672554b26719 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2838edd703d11b41f328ed4e8940391a for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dell.lan - Linux fedora 26, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/1d726c4c2c560e561cff8aa44f47df06 for a full test report.

File: easybuild/easyconfigs/r/Ruby/Ruby-2.4.2-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://cache.ruby-lang.org/pub/ruby/']
```
Comment (verdurin) [inline easybuild/easyconfigs/r/Ruby/Ruby-2.4.2-foss-2017b.eb]: @boegel - missing checksum?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dell.lan - Linux fedora 26, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/c0577934ecb837f03546f5ddbfd77342 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**